app/src/main/java/practicum/getfitla_v3/MainActivity.java

```diff
@@ -37,3 +37,6 @@ public void onClick_Nutrition(View view) {
 
     }
 }
+
+//https://stackoverflow.com/questions/28572302/android-studio-not-recognizing-classes-after-power-failure?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa
+//if android studio runs and everything turns red where it constantly says Setup JDK 
```

Comment:
Again, either put this at the top of the file, or put it in the installing/running instructions.